src/sidebar/store/modules/drafts.js

```diff
@@ -18,19 +17,29 @@ const initialState = [];
 /** @typedef {typeof initialState} State */
 
 /**
- * Helper class to encapsulate the draft properties and a few simple methods.
- *
- *  A draft consists of:
+ * @typedef {Pick<Annotation, 'id'|'$tag'>} AnnotationID
```

Comment:
Oh! I don't know that I'd think to use `Pick` here. I probably would have done something more clunky like `{Annotation['id']|Annotation['$tag']}` (assuming that works). It's good to keep seeing examples...